src/demo/polymer-lint.ts

```diff
@@ -40,7 +40,7 @@ async function getWarnings(analyzer: Analyzer, localPath: string):
     Promise<Warning[]> {
       try {
         const document = await analyzer.analyze(localPath);
-        return document.getWarnings();
+        return document.getWarnings({lookInDependencies: false});
```

Comment:
`deep` suggests that it might look inside `<template>`s.

`imported` is punchy and clear. I like it.